source/2019-07-12-the-ember-times-issue-106.md

```diff
@@ -42,12 +43,13 @@ What’s even better is that since this is licensed under the BSD-2-Clause, you
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [Updates To Ember Igniter Articles 🔥](https://emberigniter.com/articles/)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@frank06](https://github.com/frank06) has recently been updating his **awesome series of articles** regarding a variety of Ember.js subjects on his blog, **Ember Igniter**.
+
+Many of the updates are aimed at brining the articles inline with newer versions of Ember.js, including Ember 3 generally and even readying them for Octane in some cases!
```

Comment:
I’ve some trouble understanding the last part of this sentence. Maybe break it up a bit.